build/Build-FinOpsPsModule.ps1

```diff
@@ -0,0 +1,105 @@
+param
+(
+    [Parameter()]
+    [string]
+    $Version,
+
+    [Parameter()]
+    [ValidateSet('alpha', 'beta')]
+    [string]
+    $PrereleaseTag
+)
+
+$rootPath = (Get-Item -Path $PSScriptRoot).Parent.FullName
+$moduleName = 'FinOpsToolkit.psm1'
+$modulePath = Join-Path -Path $rootPath -ChildPath 'src\powershell'
+$privatePath = Join-Path -Path $modulePath -ChildPath 'private'
+$publicPath = Join-Path -Path $modulePath -ChildPath 'public'
+$releasePath = Join-Path -Path $modulePath -ChildPath 'FinOpsToolkit'
+
+if ($Version)
+{
+    if ($PrereleaseTag)
+    {
+        $versionString = "$Version-$PrereleaseTag"
+    }
+    else
+    {
+        $versionString = $Version
+    }
+
+    $releasePath = Join-Path -Path $releasePath -ChildPath $versionString
+}
+else
+{
+    $releasePath = Join-Path -Path $releasePath -ChildPath '0.0.1'
+}
+
+$psm1Path = Join-Path -Path $releasePath -ChildPath $moduleName
+$manifestPath = Join-Path -Path $releasePath -ChildPath 'FinOpsToolkit.psd1'
+
+$null = New-Item -Path $releasePath -ItemType 'Directory'
+
+'#region Private' | Add-Content -Path $psm1Path
```

Comment:
I did some light research and StringBuilder is faster than Add-Content. Should we use StringBuilder here? I used it in `Build-Bicep.ps1`.